biogeophys/FatesHydroWTFMod.F90

```diff
@@ -847,21 +888,21 @@ function th_from_psi_tfs(this,psi) result(th)
     if(psi>this%psi_max) then
 
         ! Linear range for extreme values
-
-        th = this%th_linear_sat(psi)
+        th = this%th_res+max_sf_interp*(this%th_sat-this%th_res) + &
```

Comment:
this looks like an alternative to the current linear extrapolation scheme.  I think this alternative should be encapsulated in a switch, and probably writtten into the th_linear_sat() routine.